CONTRIBUTING.md

```diff
@@ -1,11 +1,11 @@
-# Contributing to OpenMMLab
+## Contributing to OpenMMLab
```

Comment:
Why?

Reply:
Changing # to ## makes us directly create a symbolic link to docs  because those articles in docs all start with ##

Reply:
Random question: what if changing ## to # for files in docs?

Reply:
it will break the structure of docs